### PR TITLE
store/tikv: update gc_worker init.

### PR DESCRIPTION
**Pull request by disksing**
Refactor gc worker to initialize configurations faster.

### CODE REVIEW

File: store/tikv/gc_worker.go [hunk 1]
```diff
@@ -152,6 +129,44 @@ func (w *GCWorker) start(ctx goctx.Context) {
 	}
 }
 
+func (w *GCWorker) createSession() {
+	for {
+		time.Sleep(time.Second)
+
+		if !w.storeIsBootstrapped() {
```
Comment (tiancaiamao) [inline store/tikv/gc_worker.go]: @hicqu Refact lazy initialize logic of domain may benefit here.

File: store/tikv/gc_worker.go [hunk 2]
```diff
@@ -152,6 +129,44 @@ func (w *GCWorker) start(ctx goctx.Context) {
...
+		log.Warnf("[gc worker] check leader err: %v", err)
+		return
+	}
+	if isLeader {
+		err = w.leaderTick(ctx)
+		if err != nil {
+			log.Warnf("[gc worker] leader tick err: %v", err)
+		}
+	} else {
+		// Config metrics should always be updated by leader.
```
Comment (tiancaiamao) [inline store/tikv/gc_worker.go]: Your comment is inconsistent with what you do. @disksing

File: store/tikv/gc_worker.go (hunk 1 above)
Comment (disksing) [inline store/tikv/gc_worker.go]: Do you have any link to that refactor?

**Comment by tiancaiamao** [comment]
LGTM

**Comment by disksing** [comment]
PTAL @AndreMouche

File: store/tikv/gc_worker.go [hunk 3]
```diff
@@ -152,6 +129,44 @@ func (w *GCWorker) start(ctx goctx.Context) {
...
+		privilege.BindPrivilegeManager(w.session, nil)
+		return
+	}
+}
+
+func (w *GCWorker) tick(ctx goctx.Context) {
+	isLeader, err := w.checkLeader()
+	if err != nil {
+		log.Warnf("[gc worker] check leader err: %v", err)
+		return
```
Comment (AndreMouche) [inline store/tikv/gc_worker.go]: Should we return error ?

Comment (disksing) [inline store/tikv/gc_worker.go]: Need to continue the loop event if we encounter an error, so just print it then go on.

**Pull request merged**